src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -521,6 +533,15 @@ internal enum TraceFormat
         // the format for backwards compatibility.
         internal String ToString(TraceFormat traceFormat)
         {
+            StringBuilder sb = new StringBuilder(255);
```

Comment:
Maybe use `StringBuilderCache`?

Reply:
Thought about that but `StringBuilderCache` allocates a new one if current is too small; and has a retained limit of 350 chars - and the stack traces are pretty big; so its either going to bloat the cache or always allocate anyway :-/